include/arch/arm/cortex_m/mpu/arm_mpu_v8m.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2018 Linaro Limited.
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Privileged No Access, Unprivileged No Access */
+//#define NO_ACCESS       0x0
+//#define NO_ACCESS_Msk   ((NO_ACCESS << MPU_RASR_AP_Pos) & MPU_RASR_AP_Msk)
+/* Privileged No Access, Unprivileged No Access */
+//#define P_NA_U_NA       0x0
+//#define P_NA_U_NA_Msk   ((P_NA_U_NA << MPU_RASR_AP_Pos) & MPU_RASR_AP_Msk)
+/* Privileged Read Write, Unprivileged No Access */
+#define P_RW_U_NA       0x0
+#define P_RW_U_NA_Msk   ((P_RW_U_NA << MPU_RBAR_AP_Pos) & MPU_RBAR_AP_Msk)
+/* Privileged Read Write, Unprivileged Read Only */
+//#define P_RW_U_RO       0x2
```

Comment:
These access permissions configuration is not applicable for ARMv8-M. I kept the commented-out code, to make it visible to the reviewing process.